File: backend/modelarium-media-service/src/main/java/app/MediaServiceApplication.java
package app;

import app.config.properties.DotenvPropertySourceInitializer;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.r2dbc.repository.config.EnableR2dbcRepositories;

@SpringBootApplication
@EnableR2dbcRepositories
public class MediaServiceApplication {
    public static void main(String[] args) {
        SpringApplication app = new SpringApplication(MediaServiceApplication.class);
        app.addInitializers(new DotenvPropertySourceInitializer());
        app.run(args);
    }
}

File: backend/modelarium-media-service/src/main/kotlin/MediaServiceApplication.kt
package org.modelarium

import io.github.cdimascio.dotenv.dotenv
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

//@ComponentScan(
//    basePackages = [""]
//)
@SpringBootApplication
class MediaServiceApplication

fun main(args: Array<String>) {
    dotenv {
        filename = ".env"
        ignoreIfMalformed = true
        ignoreIfMissing = true
        systemProperties = true
    }
    runApplication<MediaServiceApplication>()
}